dist/assembly.xml

```diff
@@ -161,6 +161,30 @@
       <useTransitiveFiltering>false</useTransitiveFiltering>
     </dependencySet>
 
+    <!-- Cassandra module -->
+    <dependencySet>
+      <outputDirectory>${hawkular.dist.zip.root.dir}/modules/org/apache/cassandra/main</outputDirectory>
+      <includes>
+        <include>org.apache.cassandra:cassandra-all:jar</include>
```

Comment:
I don't understand why a component other than inventory needs cassandra-all. A module for the driver, why not. But cassandra-all, it's another story.